bc_obps/registration/models.py

```diff
@@ -104,6 +104,17 @@ class Meta:
 class UserAndContactCommonInfo(models.Model):
     """User and contact common information abstract base class"""
 
+    class Roles(models.TextChoices):
+        CAS_ADMIN = "cas_admin"
+        CAS_ANALYST = "cas_analyst"
+        CAS_PENDING = "cas_pending"
+        INDUSTRY_USER = "industry_user"
+        INDUSTRY_USER_ADMIN = "industry_user_admin"
+
+    class IdPs(models.TextChoices):
+        IDIR = "idir"
+        BCEIDBUSINESS = "bceidbusiness"
```

Comment:
Thanks, any suggestions for a better design?

Reply:
I'm still learning django, so I don't have suggestions, sorry! Path of least resistance is probably just to delete this here and put the check back the way it was

Reply:
Enum looks good to me! Though I think we actually could use the role choices in the User model (I'm not sure why it isn't already like that, maybe we thought we'd have a ton of roles?).
It could look like this if you wanted to do it:
```
class User(UserAndContactCommonInfo):
    """User model"""
    class Roles(models.TextChoices):
        CAS_ADMIN = "cas_admin"
        ...
...
    app_role = models.ForeignKey(
        AppRole,
        choices=Users.choices,
        on_delete=models.DO_NOTHING,
        related_name="users",
        db_comment="The role assigned to this user which defines the permissions the use has.",
    )
d

Reply:
👌
Added ticket [428](https://app.zenhub.com/workspaces/team-moose-sprint-board-60d216175a2689000ec774d0/issues/gh/bcgov/cas-registration/428)